src/main/resources/i18n/messages.properties

```diff
@@ -561,3 +561,5 @@ reviewScore.two=2
 reviewScore.one=1
 UIDNotExecuted=UID binary could not be executed. It might have been removed, not found or inaccessible.
 replay.copyLink=Copy link
+game.ended=The game "{0}" ended
+game.showInfo=Show information
```

Comment:
We don't just want to let users know about it with a boring message but instead encourage them to rate the game. We need to sell it. I'm no marketing guy, but how about this:

`Did you enjoy "<title>"? Let other people know!`  `[Rate it now]`